misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/CuratorFrameworkProvider.kt

```diff
@@ -12,6 +12,15 @@ internal class CuratorFrameworkProvider @Inject internal constructor(
 ) : Provider<CuratorFramework> {
 
   override fun get(): CuratorFramework {
+    if (config.cert_store != null && config.trust_store != null) {
+      System.setProperty("zookeeper.clientCnxnSocket", "org.apache.zookeeper.ClientCnxnSocketNetty")
+      System.setProperty("zookeeper.client.secure", "true")
+      System.setProperty("zookeeper.ssl.keyStore.location", config.cert_store.resource)
+      System.setProperty("zookeeper.ssl.keyStore.password", config.cert_store.passphrase ?: "changeit")
```

Comment:
I don't think we should promote this as a default value in misk, even though we do it in SQ.

Reply:
Why not? This is java's default password: https://docs.oracle.com/cd/E19798-01/821-1841/gjrgy/

Reply:
but is that really a best practice default? at the very least if we go this route, we should clearly document why we believe `changeit` is fine as default. at first glance to a developer it would seem extremely insecure. for example it is recommended to only us `changeit` if you store your keystore (or keystore contents)  in a secure manner and a password would just be redundant anyways.

Reply:
Oh, I misunderstood what your initial question was in reference to. Yeah, I'll just move this to the config file itself and not provide a default here.

Reply:
(Default here being a default in Misk, leaving the client to specify what password they want via their config)